content/docs/self-hosted-runners.md

```diff
@@ -207,25 +245,52 @@ Click below to see credentials needed for supported cloud service providers.
 
 </details>
 
-## Using on-premise machines as self-hosted runners
+<details>
+
+#### GCP
```

Comment:
@julieg18 is there any way to get this pattern to work with `#ids`? i.e `[link](/doc/self-hosted-runners#gcp)`

Reply:
Not in our current version of the doc-engine. Headers inside the details block work but not the details's header itself. We currently take off the auto link headers inside the details headers since it doesn't work correctly. But looking at the html and css, it may be possible to implement this if we wanted 🤔

Reply:
would be nice; but #70 is much more urgent :)